packages/website/pages/stats.js

```diff
@@ -1,7 +1,9 @@
 import { useEffect, useState } from 'react'
 import { TrustedBy } from '../components/trustedByLogos'
 import fs from 'fs'
-import { formatBytes, calculateStats } from '../lib/statsUtils'
+import decorateAdditionalCalculatedValues, {
```

Comment:
This line causing build failure. No default export from "statsUtils"

Reply:
referenced main to make sure it matched.